service/src/main/java/com/codeforcommunity/processor/authenticated/ProtectedSchoolProcessorImpl.java

```diff
@@ -1,5 +1,6 @@
 package com.codeforcommunity.processor.authenticated;
 
+import static org.jooq.generated.Tables.*;
```

Comment:
Same thing as the previous comment. I think it is better to only import the tables that we will be using and not just import all of them

Reply:
https://www.jetbrains.com/help/idea/creating-and-optimizing-imports.html#disable-wildcard-imports